hadoop-yarn-project/hadoop-yarn/hadoop-yarn-applications/hadoop-yarn-services/dev-support/findbugs-exclude.xml

```diff
@@ -56,4 +56,9 @@
       </Or>
       <Bug pattern="SE_BAD_FIELD" />
   </Match>
+  <Match>
+      <Class name="org.apache.hadoop.yarn.service.ServiceScheduler$1"/>
+      <Method name="load"/>
+      <Bug code="NP" pattern="NP_NONNULL_RETURN_VIOLATION"/>
```

Comment:
Why not delete @Nonnull

Reply:
@zeekling The reason for modifying this PR is that SpotBugs flagged the following issue.

```
NP	org.apache.hadoop.yarn.service.ServiceScheduler$1.load(ConfigFile) may return null, but is declared @Nonnull

[Bug type NP_NONNULL_RETURN_VIOLATION (click for details)](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-7019/22/artifact/out/branch-spotbugs-hadoop-yarn-project_hadoop-yarn-warnings.html#NP_NONNULL_RETURN_VIOLATION)
In class org.apache.hadoop.yarn.service.ServiceScheduler$1
In method org.apache.hadoop.yarn.service.ServiceScheduler$1.load(ConfigFile)
At ServiceScheduler.java:[line 558]
```

The `@Nonnull` annotation is from the Guava package, so we can't modify it. However, after carefully reviewing the code, adding a rule is a more straightforward solution. From my personal perspective, this change is acceptable.

Reply:
> reviewing

i see.

Reply:
@zeekling Do you have any other feedback or suggestions regarding this PR? 
cc: @zhtttylz